README.md

```diff
@@ -1,13 +1,14 @@
 # Project Movies
 
-Replace this readme with your own information about your project.
```

Comment:
Great job on a hard project to do alone! I mentioned a few comments below one other is just the background on the tv shows could be the same setup as the background of the movies for consistency. Other than that, great job on this project!